pkg/cmd/issue/view/view.go

```diff
@@ -133,16 +163,21 @@ func printHumanIssuePreview(io *iostreams.IOStreams, issue *api.Issue) error {
 
 	// Header (Title and State)
 	fmt.Fprintln(out, cs.Bold(issue.Title))
-	fmt.Fprint(out, issueStateTitleWithColor(cs, issue.State))
-	fmt.Fprintln(out, cs.Gray(fmt.Sprintf(
-		" • %s opened %s • %s",
+	fmt.Fprintf(out,
+		"%s • %s opened %s • %s\n",
+		issueStateTitleWithColor(cs, issue.State),
 		issue.Author.Login,
 		utils.FuzzyAgo(ago),
 		utils.Pluralize(issue.Comments.TotalCount, "comment"),
-	)))
+	)
+
+	// Reactions
+	if reactions := issue.ReactionGroups.String(); reactions != "" {
+		fmt.Fprint(out, reactions)
+		fmt.Fprintln(out)
```

Comment:
I'm not sure I like the current reactions rendering on its own line. Random emoji on its own line looks weird without context. (In the web UI, their positioning and rendering communicates their context sufficiently.)

For readability, could we append reactions on the status line, preferably prefixing them with "reactions"? E.g.
```
mislav (member) • Sep 18, 2020  • reactions: 👍 4, 👎 12, 🚀 1
```

Reply:
I think the main argument for having them to their own line is that they can make the current line significantly longer depending on the variety and number of reactions. 

I do like adding in the `reactions:` portion though as that aligns with what we do for other metadata fields.

Reply:
I'd prefer to keep this as is, if you both don't mind! I think it's a reasonable to assume these are the reactions without explicitly stating it here. Happy to reassess if we receive user feedback about it!